src/services/tours/tours-service.ts

```diff
@@ -58,11 +58,22 @@ export default class ToursService extends APIService {
     return this.handleTourUpdateResult(result, tour, localTour)
   }
 
-  public async delete (id: string): Promise<SingleApiResponse<void>> {
-    return this.fetchSingleDataFromApi(
+  public async delete (id: string): Promise<SingleApiResponse<unknown>> {
+    const result = await this.fetchSingleDataFromApi(
       this.getRequestUrl(this.prefix, id),
       this.getRequestBody('DELETE', {})
     )
+    return this.handleTourDeleteResult(result, id)
+  }
+
+  private async handleTourDeleteResult (result: SingleApiResponse<unknown>, id: string): Promise<SingleApiResponse<unknown>> {
```

Comment:
@Saela Do we use `unknown` because the response can be anything? Can't we typehint that?

Reply:
@Tugark done in pr #94